README-sitedev.md

```diff
@@ -1,41 +1,21 @@
-This documentation is for developers of this prototype OBO Foundry site.
+# Site Development
 
-Note this document is in markdown and is best viewed on GitHub.
+This documentation is for developers of this prototype OBO Foundry site.
 
 ## Getting Started
 
-You will first need jekyll, which can be installed by the ruby [gem](https://rubygems.org/):
-
-    gem install jekyll
-
-After checking out the code from GitHub, you can start a local server:
-
-    jekyll serve
-
-(this must always be done from the top level)
-
-Then view http://127.0.0.1:4000
-
-You can make changes locally (you will need to start jekyll again - no
-hot fixes).
-
-If you commit and push, your change will be visible within a few
-minutes on: http://obofoundry.github.io
-
-If this site becomes official we may want to institute policies for
-the site: e.g. major new changes happen on forks/branches, with a
-voting policy for merging these in.
-
-### Serving with Docker
-
 Because Jekyll can be difficult to install, Docker provides an
-alternative for running the `serve` command:
+alternative for running the `serve` command, then open http://127.0.0.1:4000:
 
 ```shell
-$ export JEKYLL_VERSION=3.5
-$ docker run --rm --volume="$PWD:/srv/jekyll" -p 4000:4000 -it jekyll/jekyll:$JEKYLL_VERSION jekyll serve
+$ docker run --rm --volume="$PWD:/srv/jekyll" -p 4000:4000 -it jekyll/jekyll:latest jekyll serve
```

Comment:
Wait are you sure jekyll:latest will work?

Reply:
Does not work for me.

Reply:
Really? I've been doing it like this. We can pin it to a certain version, I just got upset that I kept having to rewrite this terrible export command since Im using fish and not bash

Reply:
Pinned to 3.5 in 31103e0